### ISSUE TITLE
Group Sort

### ISSUE THREAD

**Issue by Becktor** [cemanil/LNets#7]
Hi,

Using your implementation of group sort I cannot get the check_group_sorted function to return 1.
I'm providing a tensor of shape (NCHW) and with a group sort of (2, axis=1).

Stepping through the implementation with a toy problem I can see that the channel layer gets sorted correctly. But the '''np.diff()''' in '''check_group_sorted''' seems to diff on a wrong channel.

Steps to reproduce
```{python}
import torch
from activations import GroupSort

tensor = torch.arange(16).roll(5).reshape(1, 4, 2, 2)

gs = GroupSort(2, axis=1)
gs(tensor)
```

If I select axis+1 in the diff it seems to be diffing the sorted values.

**Comment by cemanil** [comment]
Hi, 

Thank you for bringing this up. There was indeed an issue with the axis used in check_group_sorted function, if one doesn't use axis==-1 for sorting. I've fixed this issue in a recent commit. Please let us know if you keep encountering similar issues in the future.  

Cem

**Comment by Becktor** [comment]
Will do!
